examples/poc/README.md

```diff
@@ -1,68 +1,55 @@
-
-
-## Overview
-
-This project contains the necessary configurations and code to set up and deploy a service using Kubernetes, Envoy, and Go. The service involves routing based on the model specified (using Open AI API format), collecting metrics, and ensuring efficient load balancing.
-
-![alt text](./envoy-gateway-bootstrap.png)
-
+- A vLLM based deployment using a custom fork, with LoRA Adapters.  ***This PoC uses a modified vLLM [fork](https://github.com/kaushikmitr/vllm), the public image of the fork is here: `ghcr.io/tomatillo-and-multiverse/vllm:demo`***. A sample deployement is provided under `./manifests/samples/vllm-lora-deployment.yaml`. 
 
 ## Quickstart
 
 ### Steps
+1. **Deploy Sample vLLM Application**
+   NOTE: Create a HuggingFace API token and store it in a secret named `hf-token` with key hf_api_token`. This is configured in the `HUGGING_FACE_HUB_TOKEN` and `HF_TOKEN` environment variables in `./manifests/samples/vllm-lora-deployment.yaml`.
 
-1. **Apply Kubernetes Manifests**
    ```bash
-   cd manifests
-   kubectl apply -f ext_proc.yaml
-   kubectl apply -f vllm/vllm-lora-service.yaml
-   kubectl apply -f vllm/vllm-lora-deployment.yaml
+   kubectl apply -f ./manifests/samples/vllm-lora-deployment.yaml
+   kubectl apply -f ./manifests/samples/vllm-lora-service.yaml
    ```
+2. **Install GatewayClass with Ext Proc**
+   A custom GatewayClass `llm-gateway` which is configured with the llm routing ext proc will be installed into the `llm-gateway` namespace. When you create Gateways, make sure the `llm-gateway` GatewayClass is used.
 
-2. **Update `ext_proc.yaml`**
-   - Ensure the `ext_proc.yaml` is updated with the pod names and internal IP addresses of the vLLM replicas. This step is crucial for the correct routing of requests based on headers.
+   NOTE: Ensure the `llm-route-ext-proc` deployment is updated with the pod names and internal IP addresses of the vLLM replicas. This step is crucial for the correct routing of requests based on headers. This won't be needed once we make ext proc dynamically read the pods.
 
-2. **Update and apply `gateway.yaml`**
-   - Ensure the `gateway.yaml` is updated with the internal IP addresses of the ExtProc service. This step is also crucial for the correct routing of requests based on headers.
-    ```bash
-   cd manifests
-   kubectl apply -f gateway.yaml
+   ```bash
+   kubectl apply -f ./manifests/installation.yaml
+   ```
+3. **Deploy Gateway**
+   
+   ```bash
+   kubectl apply -f ./manifests/samples/gateway.yaml
    ```
 
-### Monitoring and Metrics
-
-- The Go application collects metrics and saves the latest response headers in memory.
-- Ensure Envoy is configured to route based on the metrics collected from the `/metric` endpoint of different service pods.
-
-## Contributing
+4. **Try it out** 
+   Wait until the gateway is ready.
+   ```bash
+   IP=$(kubectl get gateway/llm-gateway -o jsonpath='{.status.addresses[0].value}')
+   PORT=8081
```

Comment:
Isn't this wrong, llm-instance-gw is listening on 8080. Am I wrong?

Reply:
Good question! Actually in the POC setup, Envoy is configured the additional 8081 port for ext proc traffic. Updated README

Reply:
Ho yes, you're right actually. Sorry, for the confusion.